test/built-ins/Symbol/not-callable.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2019 Aleksey Shvayka. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-properties-of-symbol-instances
+description: >
+  Symbol primitives and objects are not callable.
+info: |
+  Properties of Symbol Instances
+
+  Symbol instances are ordinary objects that inherit properties from the
+  Symbol prototype object. Symbol instances have a [[SymbolData]] internal slot.
+  The [[SymbolData]] internal slot is the Symbol value represented by this
+  Symbol object.
+features: [Symbol]
+---*/
+
+assert.throws(TypeError, function() {
+  Symbol('desc')();
```

Comment:
if you store the symbol before this function, you avoid a false positive if the impl does not support symbols yet.

So:

```js
var s = Symbol();
assert.throws(TypeError, function() {
  s();
});
```

Reply:
134e812